mne/stats/regression.py

```diff
@@ -275,24 +271,30 @@ def solver(X, y):
     return evokeds
 
 
-def _prepare_rerp_data(raw, events, picks=None, decim=1):
+def _prepare_rerp_data(raw, events, picks=None, decim=1, sfreq=None):
     """Prepare events and data, primarily for `linear_regression_raw`. See
     there for an explanation of parameters and output."""
```

Comment:
I'm a bit inclined to rename all the XDawn variables that sound like MNE instances - raw, evoked etc - but are actually just np arrays, because all of these fooled me when doing this.